html/controls/operator.js

```diff
@@ -1078,6 +1084,13 @@ function createJamDialog() {
 					row.find("td.Duration").text(_timeConversions.msToMinSec(v));
 				}
 			}
+			if (key == 'PeriodClockElapsedEnd') {
+				if (WS.state[prefix + '.WalltimeEnd'] == 0 && WS.state[prefix + '.WalltimeStart'] > 0) {
+					row.find("td.PC").text("running");
+				} else {
+					row.find("td.PC").text(_timeConversions.msToMinSec(v));
```

Comment:
This doesn't seem to be working, it's always empty.

Reply:
This is showing elapsed time, we probably want whatever the operator is seeing in terms of inversion.

Reply:
True. Changed. (Also unit tests in.)